spec/system/casa_org/edit_spec.rb

```diff
@@ -102,4 +102,14 @@
     click_on "Submit"
     has_checked_field? "Show driving reimbursement"
   end
+
+  it "can upload a logo image" do
```

Comment:
Since you have more than 1 expectation for `it` block :D

```suggestion
  it "can upload a logo image", :aggregate_failure do
```

Reply:
Very nice! I didn't know about that one